src/components/audio-trimmer/audio-selector.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import React from 'react';
```

Comment:
It might be worth investigating if we can re-combine this with the audio trimmer component and just control the positive/negative space with like a boolean flag. The rest of the code is very similar.

Reply:
I explored this a bit and realized that the trimmer and the selector have slightly different nesting structures: the trimmer has two separate background masks, with the stripes, for the start and end, and those contain the handles, but the selection is all inside a single box. So they can't be simply combined with a bunch of ternaries, as far as I can tell. I think next I will try extracting the handles as a component instead, and keeping the trimmer and selector components separate.

Reply:
Okay, I extracted components for selection handles and the playhead, reducing the duplication between the trimmer and the selector